### PR TITLE
[Feat.] Added Support for ConfigMap Builder

### PR DESCRIPTION
**Pull request by rahulchheda**

### CODE REVIEW

File: kubernetes/configmap/build.go [hunk 1]
```diff
@@ -0,0 +1,53 @@
+package configmap
+
+import (
+	"errors"
+	"fmt"
+	//templatespec "github.com/litmuschaos/kube-helper/kubernetes/podtemplatespec"
+	//batchv1 "k8s.io/api/batch/v1"
```
Comment (chandankumar4) [inline kubernetes/configmap/build.go]: If not using, Can you remove this

Comment (rahulchheda) [reply kubernetes/configmap/build.go]: Sure! @chandankumar4

**Review by chandankumar4 — APPROVED** [review]

**Comment by chandankumar4** [comment]
@rahulchheda Can you add the license in the file, You can take reference from the PR: #9

File: kubernetes/configmap/configMap.go [hunk 2]
```diff
@@ -0,0 +1,10 @@
+package configmap
+
+import (
+	corev1 "k8s.io/api/core/v1"
+)
+
+// ConfigMap is a struct with contains th*corev1.ConfigMap
+type ConfigMap struct {
+	object *corev1.ConfigMap
```
Comment (Sanjay1611) [inline kubernetes/configmap/configMap.go]: Change object -> Object

**Comment by rahulchheda** [comment]
> @rahulchheda Can you add the license in the file, You can take reference from the PR: #9

Sure, would do it till EOD!

**Comment by ksatchit** [comment]
Thanks @rahulchheda - looks good! Can we add description on the PR and an issue to reference for this.

**Review by ksatchit — APPROVED** [review]

**Pull request closed without merging**